### PR TITLE
xoq

### PR DESCRIPTION
**Pull request by plosky1**
xoq

thanks to @brndd @Zth-- @ whatever deadharolds github name is

without brndd and zth's code this would have taken 10x longer
without deadharold I would have had to edit way more shit by myself

there's probably stuff me and harold missed, but it ///should/// be playable

:cl:
 * rscadd: xoq

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
Lived eht fo nwaps yawa og

**Comment by DrSnips** [comment]
Can you post what the fuck it means. That CL means nothing.

**Comment by ghost** [comment]
XOQ

**Comment by SkowronX** [comment]
box but it's 150cc mirror mode

**Comment by kilozombie** [comment]
It's xoq

**Comment by plosky1** [comment]
Think about the word box, then think about the word xoq

**Comment by PrimeDSS13** [comment]
plosky this reminds me of your unban appeal, please add images and actual text.
very exciting though

**Comment by SonixApache** [comment]
XOQ

**Comment by N3X15** [comment]
It's a hard X.

File: maps/xoq.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#ifndef MAP_OVERRIDE
+//**************************************************************
+// Map Datum -- xoqstation
+//**************************************************************
+
+/datum/map/active
+	nameShort = "xoq"
+	nameLong = "noitatS xoq"
```
Comment (Exxion) [inline maps/xoq.dm:8]: One of these words is rotated, one is reversed
It's fucked up

Comment (plosky1) [reply maps/xoq.dm:8]: well I cant rotate station OKAY, I'M NOT A GOD

Comment (brndd) [reply maps/xoq.dm:8]: uoıʇɐʇs

Comment (plosky1) [reply maps/xoq.dm:8]: doesn't work in the .dm file sadly, just gives ??

Comment (Exxion) [reply maps/xoq.dm:8]: It would in 513 but sadly we haven't updated yet

**Comment by Kurfursten** [comment]
As a general reminder, this would require Pomf to handle the binary stuff on his end

**Pull request merged**